tabs/pid_tuning.js

```diff
@@ -1,19 +1,23 @@
 'use strict';
 
 TABS.pid_tuning = {
+    RATE_PROFILE_MASK: 128,
     showAllPids: false,
-    profileDirty: false,
-    loadingProfile: true
+    updating: true,
+    dirty: false,
+    currentProfile: null,
+    currentRateProfile: null
 };
 
 TABS.pid_tuning.initialize = function (callback) {
     var self = this;
+
     if (GUI.active_tab != 'pid_tuning') {
         GUI.active_tab = 'pid_tuning';
     }
 
-    // requesting MSP_STATUS manually because it contains CONFIG.profile
-    MSP.promise(MSP_codes.MSP_STATUS).then(function() {
+    // requesting MSP_STATUS_EX manually because it contains CONFIG.profile
+    MSP.promise(MSP_codes.MSP_STATUS_EX).then(function() {
```

Comment:
It is this line that makes it not working on 2.9. 
MSP_STATUS_EX seems to be broken for 2.9, 
https://github.com/betaflight/betaflight/blob/v2.9.0/src/main/io/serial_msp.c#L742-L754
It tries to send 12 bytes but only 11 is sent.